src/rgw/rgw_sal_motr.cc

```diff
@@ -4834,6 +4834,17 @@ int MotrStore::store_email_info(const DoutPrefixProvider *dpp, optional_yield y,
   return rc;
 }
 
+int MotrStore::list_gc_objs(const DoutPrefixProvider *dpp,
+              std::vector<std::unordered_map<std::string, std::string>>& gc_entries)
+{
```

Comment:
Hi, dpp is needed I think.
```suggestion
int MotrStore::list_gc_objs(std::vector<std::unordered_map<std::string, std::string>>& gc_entries)
{
```

Reply:
You mean not needed right :D 
Removed dpp.

Reply:
yeah 😅. Thanks